codes/classical/bits/elliptic.yml

```diff
@@ -0,0 +1,59 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+
+
+code_id: 'elliptic'
```

Comment:
I think you have enough here for an 'Evaluation AG code' entry as well as the more specific 'Elliptic code' entry. Let's try the following:

1. Make an entry code_id:evaluation (parent:ag; I've now added such an entry), put any aspects associated with this general class into that entry. Make evaluation a parent of code_id:generalized_reed_solomon.
2. Add a second entry for code_id:elliptic (parent:evaluation), put any specific statements about that code there (e.g., "Evaluation AG codes on genus 1 surfaces.").

Reply:
See also http://mint.sbg.ac.at/desc_CGoppa.html